clang/lib/Lex/LiteralSupport.cpp

```diff
@@ -146,6 +144,8 @@ static unsigned ProcessCharEscape(const char *ThisTokBegin,
   // that would have been \", which would not have been the end of string.
   unsigned ResultChar = *ThisTokBuf++;
   char Escape = ResultChar;
+  bool Translate = true;
```

Comment:
```suggestion
  bool Transcode = true;
```

I would prefer this defaults to false